config/nginx.conf.erb

```diff
@@ -64,6 +64,11 @@ http {
 			proxy_pass http://app_server;
 		}
 
+		# Just in case, only forward "/policies" to Ember for a moment
+		location = /policies {
+			proxy_pass http://localhost:9000;
+		}
```

Comment:
The template should test `ENV['USE_FASTBOOT']` here, right?

Reply:
Right! Not so sure how did I miss it...